unlock-app/src/pages/_app.js

```diff
@@ -8,7 +8,7 @@ import { createUnlockStore } from '../createUnlockStore'
 import GlobalStyle from '../theme/globalStyle'
 import { ConfigContext } from '../utils/withConfig'
 
-import WalletCheckOverlay from '../components/interface/FullScreenModals'
+import FullScreenModal from '../components/interface/FullScreenModals'
```

Comment:
This is just a name change to keep it consistent with its more general purpose nature now